doc/api/cli.markdown

```diff
@@ -0,0 +1,163 @@
+# Command Line Options
+
+<!--type=misc-->
+
+Node.js comes with a wide variety of CLI options. These options expose built-in
```

Comment:
`CLI options -> command line options` perhaps? Are we sure all the users are familiar with the term? Windows users in particular...